src/handlers/userStore.ts

```diff
@@ -0,0 +1,124 @@
+import { Subscription } from "nats";
+
+import { jsonCodec, NatsHandler } from "../nats";
+import { getClient } from "../elasticSearch";
+import { ELASTICSEARCH_INDEX_NAME } from "../config";
```

Comment:
let's have the config file create a name for the `user` index and retrieve it here using `{ INDEXES.USERS }`